gr-blocks/lib/wavfile_sink_impl.cc

```diff
@@ -49,7 +49,7 @@ wavfile_sink_impl::wavfile_sink_impl(const char* filename,
 {
     int bits_per_sample;
 
-    if (n_channels > 24) {
+    if (n_channels > MAX_CHANNELS) {
         throw std::runtime_error("Number of channels greater than 24 not supported.");
```

Comment:
If you're moving out the contants then this error message should use it too. To avoid adding boost stuff I'd use ` + std::to_string(s_max_channels) +`